lib/certlint/cablint.rb

```diff
@@ -255,8 +255,8 @@ def self.lint(der)
         unless subjattrs.include? 'serialNumber'
           messages << 'E: EV certificates must include serialNumber in subject'
         end
-        unless subjattrs.include? 'L'
-          messages << 'E: EV certificates must include localityName in subject'
+        if !(subjattrs.include? 'L') && !(subjattrs.include? 'ST')
+            messages << 'E: EV certificates must include either localityName or stateOrProvinceName'
```

Comment:
For consistency, I think that error message should retain the " in subject" suffix.